src/NLog.Web/README.md

```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
```

Comment:
Think this bullet should become a short introduction (below the headline, without being a bullet)

Reply:
> Integrates NLog with the System.Web.HttpContext, and providing features like: